src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,88 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto(int... numbers) {
+        this(createNumbers(intToList(numbers)));
+    }
+
+    public Lotto(String... numbers) {
+        this(createNumbers(stringToList(numbers)));
+    }
+
+    public Lotto(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    public static Lotto of(List<Integer> numbers) {
+        return new Lotto(createNumbers(numbers));
+    }
+
+    private static List<LottoNumber> createNumbers(List<Integer> numbers) {
+        List<LottoNumber> list = new ArrayList<>();
+        for (Integer number : numbers) {
+            list.add(new LottoNumber(number));
+        }
+        return list;
+    }
+
+    private static List<Integer> stringToList(String... numbers) {
+        List<Integer> list = new ArrayList<>();
+        for (String number : numbers) {
+            list.add(Integer.parseInt(number));
+        }
+        return list;
+    }
+
+    private static List<Integer> intToList(int... numbers) {
+        List<Integer> list = new ArrayList<>();
+        for (int number : numbers) {
+            list.add(number);
+        }
+        return list;
+    }
+
+    public int matchedCount(Lotto lotto) {
```

Comment:
```suggestion
    public LottoRank matchedCount(Lotto lotto) {
```
int 보다 enum도 추가했는데 등수를 반환하면 어떨까?
이에 따른 메서드 이름도 ai와 함께 고민해볼 것을 추천